google_shopping_functions.js

```diff
@@ -5,26 +8,78 @@ var getItemsCount = function(itemData){
 	return itemData.items.length;
 };
 
-//
-//DEFINE FUNCTIONS HERE
-//
+// Question 1
+var getItems = function(objectData) {
+	return objectData.items;
+}
 
 
+console.log(getItems(data));
 
+//Question 2
 
+var items = getItems(data); 
+var getItemsbyBrand = function(items, brand) {
+	var brandItems = [];
+	for (var i = 0; i < data["items"].length; i++ ) {
+	if (data["items"][i].product.brand === brand) {
+		brandItems.push(items[i]); 
+	}
+}
+	return brandItems;
+}
 
-//load products.json
-var data = require("./products.json")
 
-//
-//USE FUNCTIONS HERE
-//
+// 
+
+// var itemsByBrand = getItemsbyBrand(items, 'Nikon');
+// console.log(itemsByBrand);
+
+//Question 3
+
+var items = getItems(data);
+var getItemsByAuthor = function(items, author) {
+	var authorItems = [];
+	for (var i = 0; i < data["items"].length; i++ ) {
+	if (data["items"][i].product.author.name.indexOf(author) !=-1) {
```

Comment:
Same thing here, use your parameter in your if statement. Also, you will need !== instead of != in your if statement, as they mean different things.

```
for (var i = 0; i < items.length; i++) {
...
if (items[i].product.author.name.indexOf(author) !== -1) {
```